src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -326,6 +326,11 @@ public void reset() {
     }
 
     @Override
+    public boolean usesCustomClient() {
+        return false;
```

Comment:
What if a forge client connects to the vanilla server? Should this remain false?